servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultKeepAlivePolicy.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.http.netty;
+
+import io.servicetalk.http.netty.H2ProtocolConfig.KeepAlivePolicy;
+
+import java.time.Duration;
+
+import static java.time.Duration.ofSeconds;
+import static java.util.Objects.requireNonNull;
+
+final class DefaultKeepAlivePolicy implements KeepAlivePolicy {
+    static final Duration DEFAULT_IDLE_DURATION = ofSeconds(30);
+    static final Duration DEFAULT_ACK_TIMEOUT = ofSeconds(30);
```

Comment:
If you will remove the default ctor, you can move these constants to `KeepAlivePolicies`.